spec/presenters/activity_presenter_spec.rb

```diff
@@ -491,14 +491,17 @@
   describe "#forecasted_total_for_date_range" do
     it "returns the planned disbursement total for a date range as a formatted number" do
       project = create(:project_activity, :with_report)
-      _disbursement_1 = create(:planned_disbursement, parent_activity: project, value: 200.20, period_start_date: Date.today)
-      _disbursement_2 = create(:planned_disbursement, parent_activity: project, value: 1500, period_start_date: 3.months.ago)
+      current_financial_quarter = Date.parse("2020-07-01")
+      last_financial_quarter = Date.parse("2020-04-01")
+      next_financial_quarter = Date.parse("2020-10-01")
+      _disbursement_1 = create(:planned_disbursement, parent_activity: project, value: 200.20, period_start_date: current_financial_quarter, financial_quarter: 2, financial_year: 2020)
+      _disbursement_2 = create(:planned_disbursement, parent_activity: project, value: 1500, period_start_date: last_financial_quarter, financial_quarter: 1, financial_year: 2019)
```

Comment:
I think I might have misunderstood something here... quick question: If `last_financial_quarter` starts on April 2020 based on your `Date.parse("2020-04-01")` Would the financial_year be 2020 then?

Reply:
No you haven't you are right! last_financial_quarter should be 1 2020. Thank you! I hope this is all a bit clearer when we can remove the dates in the next work.

Reply:
Yes, it is clear now. I was a bit confused with so many dates and quarters. Thanks for the [other explanation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/708#discussion_r516524462) too. It made clear why we need dates now that we won't need later :-)